crates/re_types/src/lib.rs

```diff
@@ -502,5 +502,7 @@ mod size_bytes;
 pub use component_name::ComponentName;
 pub use size_bytes::SizeBytes;
 
+pub mod arrow_adapter;
```

Comment:
Only expose `ArrowString` itself so the details of the module hierarchy don't become part of the public API.

```suggestion
mod arrow_adapter;
pub use self::arrow_adapter::ArrowString;